Web3/Classes/Core/Web3/Web3.swift

```diff
@@ -278,6 +329,24 @@ public struct Web3 {
 
             properties.provider.send(request: req, response: response)
         }
+        
+        public func sendTransaction(
+            transaction: EthereumTransaction,
+            response: @escaping Web3ResponseCompletion<EthereumData>
+        ) {
+            guard transaction.from != nil else {
+                let error = Web3Response<EthereumData>(error: .requestFailed(nil))
```

Comment:
Is this the best way we can handle this case? What would an Ethereum node return? Maybe we shouldn't check it locally...

Reply:
This was borrowed from what web3.js does. It would be good to at least have a more specific error for that case.

Reply:
Yes I am happy with a more specific error. I didn't know the web3.js implementation...

Reply:
One thing that web3js also does here is store a currentAddress that gets populated if from is empty. Might be worth borrowing that convention as well, but perhaps for another PR as this one is getting quite big